koku/api/metrics/constants.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Model for our cost model metric map."""
+OCP_METRIC_CPU_CORE_USAGE_HOUR = "cpu_core_usage_per_hour"
```

Comment:
I would consider doing 1 of 2 things. 

1) Move these constant definitions to the `__init__.py` for metrics so they are importable from metrics or 
2) Alias the import `from api.metrics import constants` to `from api.metrics import constants as metric_constants`

I lean slightly towards 1, but either way the goal is improved readability/understandability in the places where you import these constants. Just importing something as "constants" doesn't provide the context that they are metric specific constants.

Reply:
I chose option #2 @adberglund , it's updated now.